src/sonic-yang-models/setup.py

```diff
@@ -82,6 +82,7 @@ def run(self):
                          './yang-models/sonic-bgp-common.yang',
                          './yang-models/sonic-bgp-global.yang',
                          './yang-models/sonic-bgp-monitor.yang',
+                         './yang-models/sonic-bgp-internal-neighbor.yang',
```

Comment:
Need to add this file to cvlyang-models

Reply:
Hi @ganglyu, currently the BGP_VOQ_CHASSIS_NEIGHBOR and BGP_INTERNAL_NEIGHBOR tables are used by bgpcfgd I am not sure if the sonic-mgmt-framework or frrcfgd will use these tables. Hence, I did not add them to the cvlyang-models